MatomoTracker/Event.swift

```diff
@@ -138,4 +138,9 @@ extension Event {
         self.orderLastDate = orderLastDate
         self.isCustomAction = isCustomAction
     }
+
+    @available(*, unavailable, renamed: "init(tracker:action:url:referer:eventCategory:eventAction:eventName:eventValue:customTrackingParameters:searchQuery:searchCategory:searchResultsCount:dimensions:variables:contentName:contentInteraction:contentPiece:contentTarget:goalID:revenue:orderID:orderItems:orderRevenue:orderSubTotal:orderTax:orderShippingCost:orderDiscount:orderLastDate:isCustomAction:)")
```

Comment:
Is there a reason why you chose `unavailable` here vs `deprecated`?

Reply:
Sorry, no, that was a leftover from work in progress. Fixed now.